lib/spotlight_search/helpers.rb

```diff
@@ -9,6 +9,14 @@ def sortable(column, title = nil, sort_column="created_at", sort_direction="asc"
       content_tag("a","Title", class: css_class, data: {sort_column: column, sort_direction: direction, behaviour: 'sort', type: 'anchor-filter'})
     end
 
+    def exportable(records, email)
```

Comment:
We cannot pass the records, It needs to apply filters as well. This needs to be in JS.

Reply:
the entire table and this link gets replaced once the filters are run right. in that case, aren't records the new set of records after filter has been applied. i am not sure though

Reply:
@sajinmp - as discussed, we will need to use the JS because of association fetching and pagination. I suggest making edits to the method that @anbublacky uses to search.

Reply:
used js to fetch filters